firebase-database/src/iosMain/c_interop/Cartfile

```diff
@@ -0,0 +1 @@
+binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseDatabaseBinary.json" == 6.30.0
```

Comment:
New line at the end will help not breaking console output when you do `cat` of this file